miden-tx/src/host/state_tracker.rs

```diff
@@ -32,15 +20,15 @@ const STORAGE_TREE_DEPTH: Felt = Felt::new(AccountStorage::STORAGE_TREE_DEPTH as
 /// - all account storage changes.
 /// - account code changes.
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct AccountDeltaTracker {
+pub struct StateTracker {
```

Comment:
As mentioned in one of the previous comments, I'm not sure this is an improvement. Maybe something like `AccountStateTracker` would be better?